jobqueue_features/tests/test_cluster.py

```diff
@@ -123,13 +131,14 @@ def test_mpi_complex_job_cluster_fail(self):
             # 24/2 = 12). For an MPI job we expect the core count (which is the total
             # number of cores to be used) to be divisible by cpus_per_tasks but that is
             # not true in this case resulting in a ValueError
-            cluster = get_cluster(
+            get_cluster(
                 queue_type="gpus",
                 mpi_mode=True,
                 cores=2,
                 ntasks_per_node=2,
                 **self.kwargs
             )
+        controller.delete_cluster("dask-worker-gpus")
```

Comment:
Why does this one have special behaviour and not use `cluster.name`?

Reply:
Well, the issue was that cluster with such name already exist but the reason was that in another test which passed successfully the created cluster was not deleted.

Reply:
Ok, understood.